settings.gradle

```diff
@@ -91,4 +91,8 @@ include 'release:docker'
 include 'release:maven'
 include 'e2e-test:peerforwarder'
 include 'rss-source'
+include 'data-prepper-plugins:json-codec'
+include 'data-prepper-plugins:parquet-codecs'
+include 'data-prepper-plugins:newline-codecs'
+findProject(':data-prepper-plugins:newline-codecs')?.name = 'newline-codecs'
```

Comment:
I'm unsure what you are trying to do here, but this does not seem right.

Reply:
`findProject(':data-prepper-plugins:newline-codecs')?.name = 'newline-codecs'`
This line gets autogenearted when we build our newline-codecs module but it is redundant and could be removed.